api-docs/list-clients.md

```diff
@@ -0,0 +1,34 @@
+# View a list of clients
+
+API is provided to query a list of clients from the database.
+
+##Request
+GET /api/clients/?firstName=Mario&lastName=Luigi&zip=84107&page=1
+
+Notes:
+
+- Result is limited to 100 rows ordered by lastName, firstName
+- Values for the query should be URL encoded
+- "page" query parameter defaults to 1
+- "firstName" and "lastName" can be partial
+- If "firstName", "lastName", and "zip" are not provided the top 100 rows will be returned ordered by lastName, firstName
+
+##Response
+
+```json
+{
+  "numClients": 1,
+  "clients": [
+    {
+      "id": 56,
+      "firstName": "Mario",
+      "lastName": "Luigi",
+      "zip": "84107",
+      "dob": "01/01/1981",
+      "primaryPhone": "5551111111",
+      "dateAdded": "04/28/2019",
```

Comment:
Our APIs should be using the standard ISO format for dates (YYYY/MM/DD), instead of MM/DD/YYYY format of date. This is because code always understands the ISO format, but can get confused between month and day if you do it MM/DD/YYYY.

See https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/3d06f711607a47c520a62cc3ec32eeaa88903c81/backend/apis/clients/add-client.api.js#L226